utils/upload_to_s3/config.yaml

```
@@ -0,0 +1,9 @@
+# leave black for boto to access your AWS config.
+aws_access_key: ""
+aws_secret_key: ""
```

not loving those blank lines.
how about this (note that I also renamed the keys names to the standard):
```suggestion
aws_access_key_id: ${env:AWS_ACCESS_KEY_ID}
aws_secret_access_key: ${env:AWS_SECRET_ACCESS_KEY}
```